### DISCUSSION TITLE
add option to create release drafts with nx release instead of publishing them directly

### DISCUSSION THREAD

**Discussion by GreedyA1** [nrwl/nx#23577] [Feature Requests]
[github.ts](https://github.com/nrwl/nx/blob/master/packages/nx/src/command-line/release/utils/github.ts#L421)
in the file 

```
export interface GithubRelease {
  id?: string;
  tag_name: string;
  target_commitish?: string;
  name?: string;
  body?: string;
  draft?: boolean;
  prerelease?: boolean;
}
```

does include draft property but it is never used.
can we get option to only create draft for release instead of publishing it.

**Comment by theerebuss** [comment]
This would be a great feature to have, we use draft releases so we can add some human readable explanation of what's new and push these changelogs to Slack for non-technical users.
Is there any plan to add this?
